index.md

```diff
@@ -6,7 +6,7 @@ redirect_from:
   - "/2019/"
 ---
 
-Our Networks 2019 explores the collective care and maintenance of alternative networking practices&mdash;new protocols, peer-to-peer connections, offline-first computing, or community-based governance and ownership. Building on last year, where we asked how to move beyond solitary practices of do it yourself (DIY), in 2019 we explore what critical and creative practices offer **RE: Infrastructures** of alternatives.
+**RE: Infrastructures** explores the collective care and maintenance of alternative networking practices—new protocols, peer-to-peer connections, offline-first computing, or community-based governance. 
```

Comment:
Minor thought: replacing _or_ with _and_?: `offline-first computing, and community-based governance`

Reply:
much better!

Reply:
```suggestion
**RE: Infrastructures** explores the collective care and maintenance of alternative networking practices—new protocols, peer-to-peer connections, offline-first computing, and community-based governance. 
```